### PR TITLE
Document walkthrough

### PR DESCRIPTION
**Pull request by b-admike**
connect to https://github.com/strongloop-internal/scrum-apex/issues/227

### CODE REVIEW

**Comment by ssh24** [comment]
Before we start reviewing, should this be on a separate personal repo of yours?

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
\ No newline at end of file
+# lb-next-app-walkthrough-with-juggler
+
+## Installation (installing juggler, and your favourite connector(s))
+Legacy juggler support has been enabled in `loopback-next` and can be imported from the `@loopback/repository` package. In order to do this, save `@loopback/repository` dependency in your application, and import` juggler` from `legacy-juggler-bridge` file [like so](https://github.com/strongloop/loopback-next-example/blob/poc/lb-next-w-juggler/services/account/repositories/account/index.ts#L1-#L2).
+You can then install your favourite connector by saving it as part of your
+app dependencies. 
+
+## Datasource Configuration (making your new datasource)
+You can Define a datasource using legacy juggler in your LoopBack-Next app as follows:
```
Comment (0candy) [inline README.md]: Why is `Define` capitalized?

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+3. [Create a new instance of `DataSourceConstructor` with your configuration object](https://github.com/strongloop/loopback-next-example/blob/poc/lb-next-w-juggler/services/account/repositories/account/index.ts#L11-#L18)
+4. [Call `createModel` method on your `DataSourceConstructor` instance and assign it to your `model` property](https://github.com/strongloop/loopback-next-example/blob/poc/lb-next-w-juggler/services/account/repositories/account/index.ts#L20)
+
+## Controller Configuration (hooking up the datasource to the controller)
+Since the datasource is defined for your repository, all the CRUD methods you call in your repository will use juggler and your connector's methods unless you overwrite them. In your controller, you will need to define a repository property and create a new instance of the repository you configred your DataSource for in the constructor of your controller class [code snippet here](https://github.com/strongloop/loopback-next-example/blob/poc/lb-next-w-juggler/services/account/controllers/AccountController.ts#L8-#L12).
```
Comment (0candy) [inline README.md]: typo `configured`

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+##Defining CRUD methods for your application
```
Comment (virkt25) [inline README.md]: Need space after ```##```

**Pull request closed without merging**